README.rst

```diff
@@ -166,6 +166,15 @@ Backup a single organization repository with everything else (wiki, pull request
     # e.g. git@github.com:docker/cli.git
     github-backup $ORGANIZATION -P -t $ACCESS_TOKEN -o . --all -O -R $REPO
 
+Testing
+=======
+
+This project currently contains no unit tests.  To run linting::
+
+    pip install flake8
+    flake8 --ignore=E501
```

Comment:
Can you add a travis or circleci config for this? Otherwise it's almost certainly going to break again.

Reply:
I can do that but you'll need to do the initial setup since I doubt I have permissions.  If some CI is hooked up, I can add a config file and be able to test it too.